src/analyze/analyze.c

```diff
@@ -274,6 +276,8 @@ static int help(int argc, char *argv[], void *userdata) {
                "                             security review of the unit(s)\n"
                "     --unit=UNIT             Evaluate conditions and asserts of unit\n"
                "     --table                 Output plot's raw time data as a table\n"
+               "  -s --scale=FACTOR          Stretch the x-axis of the plot by FACTOR (default: 1.0)\n"
```

Comment:
shorthands are prime real estate, and sd-analyze has tons of options and will get more. This is not something common enough to warrant taking over one of the 52 available single-letter options, so please just use the long one